Sources/MapboxMaps/Gestures/GestureHandlerDelegate.swift

```diff
@@ -74,7 +74,7 @@ internal extension GestureHandlerDelegate {
 
     func scaleForZoom() -> CGFloat { return 0.0 }
 
-    func pinchScaleChanged(with newScale: CGFloat, andAnchor anchor: CGPoint) {}
+    func pinchChanged(with zoom: CGFloat, anchor: CGPoint, offset: CGSize) { }
```

Comment:
We should delete this entire extension on GestureHandlerDelegate. I don't see a reason for it given that it's an internal construct.

Reply:
thats fine with me. To be honest, I was simply following the pre-established pattern here. I think a few of these default implementations are actually used so I will move them into GestureManager.